Makefile

```diff
@@ -82,6 +82,12 @@ docker/build:     ## Build all development images.
 docker/test:      ## Run unit tests.
 	$(call exec_or_run, api, $(DJ_MANAGER), test, galaxy_ng.tests.unit$(TEST))
 
+# e.g: make docker/test FLAGS="my pytest flags"
+.PHONY: docker/test-integration
+docker/test-integration:      ## Run integration tests.
+	docker build . -f dev/standalone/integration-test-dockerfile -t galaxy-integration-runner
```

Comment:
do we need to rebuilt this every time the tests run?

Reply:
@rochacbruno yeah, the files aren't mounted into the container. As long as docker is caching things properly, this should be pretty fast. We don't need to do it this way. We could mount the `galaxy` folder and provide a `make build/integration-container` command to build the image and a `make docker/test-integration` command to run it, but this seems like it works well enough.